### ISSUE TITLE
how to install AdNauseam for development 

### ISSUE THREAD

**Issue by chirag127** [dhowe/AdNauseam#2141]
### Describe the issue

how to install AdNauseam for development.

I'm getting following error 
![image](https://user-images.githubusercontent.com/76880977/180044520-3ffb15ed-a9a5-4ba5-8ba2-e4dddd910845.png)

### Screenshot in which the issue can be seen

![image](https://user-images.githubusercontent.com/76880977/180044520-3ffb15ed-a9a5-4ba5-8ba2-e4dddd910845.png)

### Steps for anyone to reproduce the issue

1. clone repo
2. drag the cloned repo

### Your settings

- OS/version: windows
- Browser/version:
![image](https://user-images.githubusercontent.com/76880977/178135116-00056c24-7521-4b2c-b0a5-36ac28a2457a.png)

**Comment by mneunomne** [comment]
Hey @chirag127, to install the dev version you need to enable "developer mode" and select "load unpacked" instead. 

You can find the documentation about installation [here](https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chromium-based-browsers#install-adnauseam)

**Comment by chirag127** [comment]
> Hey @chirag127, to install the dev version you need to enable "developer mode" and select "load unpacked" instead.
> 
> You can find the documentation about installation [here](https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chromium-based-browsers#install-adnauseam)

Hey @mneunomne,

Thanks for your reply. Are you asking me to download the release? I'm going to develop the extension for making the pull request that @dhowe  requested in an issue I made. I will be using edge and this is my first time doing this.
I followed the instructions on the website, but when I try to load unpacked from my fork I'm still getting the same error message .

Can you help me with this?

Thanks,

**Comment by chirag127** [comment]
I have just changed 
![image](https://user-images.githubusercontent.com/76880977/180048147-1082a22c-ca57-4666-98b4-80d7c9fa039d.png)

and I want to test it

**Comment by chirag127** [comment]
seems like the main branch is not stable

**Comment by chirag127** [comment]
I followed https://github.com/dhowe/AdNauseam/wiki/Building-AdNauseam-from-source-(for-developers) but still getting the errors

**Comment by mneunomne** [comment]
For me it is working fine. What errors are you getting?

**Comment by mneunomne** [comment]
@chirag127 , any success?

**Comment by chirag127** [comment]
> @chirag127 , any success?

Nope, left trying. Just waiting for you guys to release officially.
